examples/scalar_data/geometry_data.py

```diff
@@ -0,0 +1,46 @@
+"""
+Associating data with geometries
+--------------------------------
+
+This example shows how to colour geometries based on a data array.  This
+functionality is available since Cartopy 0.23.
```

Comment:
Is there a way to do a `.. since:` or something to automatically tag this with sphinx-gallery. I like that you put this in there to let people know when it came in!

Reply:
I don't know if there is but I am far from an expert in sphinx and sphinx-gallery.  Over at https://github.com/matplotlib/matplotlib/issues/27292 Hannah was advocating using sphinx tags for that sort of thing.